addons/ofxOsc/libs/oscpack/src/ip/posix/UdpSocket.cpp

```diff
@@ -502,7 +502,7 @@ class SocketReceiveMultiplexer::Implementation{
                 if( FD_ISSET( breakPipe_[0], &tempfds ) ){
                     // clear pending data from the asynchronous break pipe
                     char c;
-                    read( breakPipe_[0], &c, 1 );
+                    if(read( breakPipe_[0], &c, 1 )) {};
```

Comment:
Uh, this looks a bit odd ... `read()` should be able to stand alone, no? Does gcc print a warning for this line?

Reply:
Yes, unused return value.

Reply:
Maybe a macro would be better.